cypress/integration/interface_builder_test.js

```diff
@@ -0,0 +1,35 @@
+describe('Login tests', () => {
+  context('no access before login', () => {
+    it('redirects to home', () => {
+      cy.visit('/interfaces/new');
+      cy.location('pathname').should('eq', '/login');
+
+      cy.visit('/interfaces/testInterface');
+      cy.location('pathname').should('eq', '/login');
+    });
+  });
+
+  context('authenticated', () => {
+    beforeEach(() => {
+      cy.login();
+      cy.fixture('test.astarte.FirstInterface').as('test_interface');
+
+      cy.server();
+      cy.route('GET', '/realmmanagement/v1/*/interfaces/*/*', '@test_interface');
+    });
+
+    it('load empty builder', () => {
```

Comment:
Where possible we could use present tense to describe desired state instead of imperative forms:
`it('load empty builder')` -> `it('loads empty builder')`
`it('show selected interface')` -> `it('shows selected interface')`
What do you think?

Reply:
It makes sense, we test for capabilities and that makes the log easier to read/understand